variables.tf

```diff
@@ -359,7 +359,7 @@ variable "netapp_network_features" {
 
 variable "node_pools_availability_zone" {
   type    = string
-  default = null
+  default = "1"
```

Comment:
Not sure this is wanted.  Setting a default conflicts with another portion of this PR

```
 zones = (var.node_pools_availability_zone == "" || var.node_pools_proximity_placement == true) ? [] : (var.node_pools_availability_zone != null) ? [var.node_pools_availability_zone] : var.node_pools_availability_zones`
 ```
 
With this in place, specifying `node_pools_availability_zones` now requires

``` 
node_pools_availability_zone=null
node_pools_availability_zones=["1", "2", "3"]

Reply:
As `node_pools_availability_zones` will be an undocumented variable, we are required to keep the default value of the variable `node_pools_availability_zone` same as before. I have adjusted the condition to honor that.